website/source/docs/agent/checks.html.md

```diff
@@ -366,3 +366,20 @@ key in your configuration file.
   ]
 }
 ```
+
+## Success/Failures before passing/critical
+
+A check may be set to become passing/critical only if a specified number of successive
+checks return passing/critical in a row. Status will stay identical as before until
+the threshold is reached.
+This feature is available for HTTP, TCP, gRPC, Docker & Monitor checks.
```

Comment:
Do you think it would be good to explicitly point out that the default is 1 for both?

Reply:
Indeed, done